resources/views/privacy.blade.php

```diff
@@ -13,29 +13,73 @@
     <div class="row">
         <div class="col-md-12">
             <p>Your use of this application is primarily governed by the <a href="http://www.gatech.edu/privacy">Georgia Tech Privacy and Legal Notice</a>. This document describes how RoboJackets will handle your information internally and under what circumstances it may be shared outside of RoboJackets.</p>
-            <p>RoboJackets may store the following information about you each time you log in, as reported by the Georgia Tech Enterprise Directory. You may opt out of this data collection by not using the application.</p>
+
+            <h2>What data does RoboJackets collect?</h2>
+            <p>RoboJackets stores the following information about you each time you log in, as reported by the Georgia Tech Enterprise Directory. You may opt out of this data collection by not using the application.</p>
             <ul>
                 <li>Your first name</li>
                 <li>Your last name</li>
                 <li>Your primary Georgia Tech email address</li>
                 <li>Your GTID number</li>
                 <li>Your Georgia Tech username</li>
+                <li>Your major</li>
+                <li>Your class standing</li>
```

Comment:
This list should include nginx IP logs (I need to check what exactly we log)

Reply:
That's technically explicitly covered by GT's policy, which we extend. But we can add it in here as well.